### PR TITLE
write js search for type drink page

### LINKED ISSUE #46: [Js] search function (typeDrink page)

### PR DESCRIPTION
**Pull request by muhannaAtalla20**
Relates #46

### CODE REVIEW

File: js/typeDrink.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const holder = document.querySelector(".cards-drink");
+/* End Selectors */
+/* Start Event Listener */  
+let final = [];
+searchInp.addEventListener("keyup", (e) => {
+    const searchString = e.target.value;
+    let filterd = final.drinks.filter((ch) => {
+        return ch.strDrink.toUpperCase().includes(searchString.toUpperCase());
+    });
+    console.log(e.key)
```
Comment (yaqootturman) [inline js/typeDrink.js]: delete console

File: js/typeDrink.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/* End Event Listener */
+/* Start Function */ 
+const loadCh = async () => {
+    try {
+    let res = await fetch(
+        "https://www.thecocktaildb.com/api/json/v1/1/filter.php?c=Cocktail"
+    );
+    final = await res.json();
+        let arr  = final.drinks;
+    // console.log(final);
```
Comment (yaqootturman) [inline js/typeDrink.js]: delete comment

File: js/typeDrink.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+};
+loadCh();
+const showMeals = (ch) => {
+    const htmlDiv = ch
+    .map((ch) => {
+        return `<div class="card">
+            <img src="${ch.strDrinkThumb}" alt="food">
+            <p>${ch.strDrink}</p>
+            <button class="get-recipe">Get Recipe</button>
```
Comment (yaqootturman) [inline js/typeDrink.js:37]: what about its functionality?

File: js/typeDrink.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    );
+    final = await res.json();
+        let arr  = final.drinks;
+    // console.log(final);
+    showMeals(arr);
+    } catch (err) {
+    console.log(err);
+    }
+};
+loadCh();
```
Comment (yaqootturman) [inline js/typeDrink.js:30]: I'm really want to know what ch refers to : (

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: js/typeDrink.js (hunk 1 above)
Comment (muhannaAtalla20) [reply js/typeDrink.js]: done

File: js/typeDrink.js (hunk 2 above)
Comment (muhannaAtalla20) [reply js/typeDrink.js]: done

File: js/typeDrink.js (hunk 3 above)
Comment (muhannaAtalla20) [reply js/typeDrink.js:37]: done

File: js/typeDrink.js (hunk 4 above)
Comment (muhannaAtalla20) [reply js/typeDrink.js:30]: done

**Review by IsraaHamdi — APPROVED** [review]

**Review by farahalzard — APPROVED** [review]

File: js/typeDrink.js (hunk 3 above)
Comment (yaqootturman) [reply js/typeDrink.js:37]: what did you mean by 'done'? I can't see its functionality

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by yaqootturman — COMMENTED** [review]
I can't remember that we talk about the functionality of button, so I will appreciate if you add it

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**